### ISSUE TITLE
Pls male a tutorial:how to add bonus weeks on Friday night funkin android 

### ISSUE THREAD

**Issue by itzmedavidmodder** [luckydog7/Funkin-android#920]
I don't know how to add weeks on android pls male a tutorial pr explain me how to
I want to make a fnf mod but with a bonus week

**Comment by ROBIKALIVE13** [comment]
> I don't know how to add weeks on android pls male a tutorial pr explain me how to
> I want to make a fnf mod but with a bonus week

u cant add weeks on android only pc

**Comment by Fixthebug307** [comment]
But why zack can make it bruh.

**Comment by Fixthebug307** [comment]
Yeah can you teach me how to make a custom week on pc with android verison.

**Comment by ROBIKALIVE13** [comment]
>  Yeah can you teach me how to make a custom week on pc with android verison.

i dont have a pc

**Comment by Fixthebug307** [comment]
.-.

**Comment by bollibee** [comment]
Maybe you should check out bbpanzu coding tutorial I know it's the pc version but I think it's the same:https://youtu.be/ac94p0zqqYM
